### PR TITLE
Created a new model using top 5 features from feature selection

### LINKED ISSUE #3: Improve the ML model based on different features as inputs and model classifier
The ML model in [model_build.ipynb](https://github.com/CMU-313/fall-2021-hw4-crabs-adjust-humidity/blob/master/dockerfile/apps/app.py) should have improved metrics beyond the baseline model provided. Different features should be experimented with, as well as different types of models. The resultant model should have a reasonably improved F1 score and should also include cross validation.

### PR DESCRIPTION
**Pull request by maggiehua01**
Modified the model ipynb file to find the top five most weighted features using sklearn's feature selection rather than randomly selecting three features. These features are as follows:
Medu (mother's education)
failures (number of past class failures)
schoolsup (extra educational support)
Dalc (workday alcohol consumption)
Walc (weekend alcohol consumption)

G1 and G2 were not considered to reflect the circumstances of application evaluation.

First separated the data into training and testing, and thus added cross validation with a test size of 0.3. Model was trained on the training data and then tested and evaluated on the testing data using the random forest classifier.

Results:
F1- 0.375
Precision- 0.375
Recall- 0.375
Accuracy- 0.8319327731092437

Resolves #3

### CODE REVIEW

**Review by mtandon18 — APPROVED** [review]
Looks good!
